sdk/include/ndk/pstypes.h

```diff
@@ -985,7 +1002,11 @@ typedef struct _EPROCESS_QUOTA_ENTRY
 
 typedef struct _EPROCESS_QUOTA_BLOCK
 {
-    EPROCESS_QUOTA_ENTRY QuotaEntry[3];
+#if (NTDDI_VERSION == NTDDI_LONGHORN)
+    EPROCESS_QUOTA_ENTRY QuotaEntry[PsQuotaTypes - 1];
+#else
+    EPROCESS_QUOTA_ENTRY QuotaEntry[PsQuotaTypes];
+#endif
```

Comment:
Why this check here? `PsQuotaTypes` would already have different value based on the enum definition.
```suggestion
    EPROCESS_QUOTA_ENTRY QuotaEntry[PsQuotaTypes];
```